test/Libraries/RevitIntegrationTests/BugTests.cs

```diff
@@ -763,6 +763,24 @@ public void CanOpenAndRunOtherFilesAfterOpeningFileWithSelectNode()
             RunCurrentModel();
         }
 
+        [Test]
+        [Category("RegressionTests")]
+        [TestModel(@".\empty.rfa")]
+        public void AllElementsInActiveViewReturnsViewableElements()
+        {
+            string filePath = Path.Combine(workingDirectory, @".\Bugs\MAGN_7641_simplified.dyn");
+            string testPath = Path.GetFullPath(filePath);
+
+            ViewModel.OpenCommand.Execute(testPath);
+            AssertNoDummyNodes();
+            RunCurrentModel();
```

Comment:
Yes, the actual cause of this defect is not quite related to view operations.

That is why I have simplified the original case. The first run is to create an import instance which is intentionally created at the center of the view. After the import instance is created, a lot of elements which are not visible will be created. The second run is to sure the node to be tested will not return these invisible elements.